packages/card/CtaCard.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { cnb } from "cnbuilder";
+import { XssSafeContent } from "@squiz/xaccel-xss-safe-content";
+import ChevronRight from "../SVG-library/ChevronRight";
+import ExternalArrowUnstyled from "../SVG-library/ExternalArrowUnstyled";
+
+/**
+ * The CTA Card component
+ * This card is used in the CTA Cards Block component
+ *
+ * @param {string} title The title of the card
+ * @param {string} eyebrow Superheading above the title
+ * @param {string} description Body text of the card
+ * @param {string} internalUrl Internal link to Matrix asset
+ * @param {string} externalUrl Manually entered external link
+ * @param {boolean} isNewWindow Whether the link should open in a new window
+ * @return {JSX.element}
+ */
+
+export default function CtaCard({
+  title,
+  eyebrow,
+  description,
+  internalUrl,
+  externalUrl,
+  isNewWindow,
+}) {
+  const hasLink = !!internalUrl || !!externalUrl;
+  // Check extenalUrl field to see if it is actually external
+  // Do not use external arrow and rel attributes if link has news.stanford.edu
+  const isRealExternalLink =
+    !!externalUrl && !externalUrl?.includes("news.stanford.edu");
+
+  return (
+    <article
+      className={cnb(
+        "su-group su-relative su-w-full su-flex su-flex-col su-break-words su-bg-white su-rounded-[8px] su-rs-pt-5 su-rs-px-4 su-max-w-900 su-mx-auto su-transition-shadow su-border su-border-black-30/30 su-shadow",
+        internalUrl || externalUrl
+          ? "hover:su-shadow-md focus-within:su-shadow-md"
+          : "",
+        hasLink ? "su-rs-pb-4" : "su-rs-pb-5"
+      )}
+    >
+      {eyebrow && (
+        <span
+          aria-hidden
+          className="su-type-1 su-text-black-60 su-font-semibold su-rs-mb-1"
+        >
+          {eyebrow}
+        </span>
+      )}
+      <h2 className="su-type-5 md:su-type-4 2xl:su-type-3 su-mb-0 su-font-sans su-text-black">
+        {hasLink ? (
+          // eslint-disable-next-line react/jsx-no-target-blank
+          <a
+            href={internalUrl || externalUrl}
+            target={isNewWindow ? "_blank" : undefined}
+            rel={isRealExternalLink ? "noopener nofollow" : undefined}
+            className="group-hocus-within:su-underline su-stretched-link su-text-black group-hocus-within:su-text-digital-red"
+          >
+            {eyebrow && <span className="su-sr-only">{`${eyebrow}:`}</span>}
+            {title}
+          </a>
+        ) : (
+          <>
+            {eyebrow && <span className="su-sr-only">{`${eyebrow}:`}</span>}
+            {title}
+          </>
```

Comment:
As mentioned above, this is where I stick the eyebrow into the h2 as invisible screen reader only text. So here it would read to the screen reader like eg "Learn: See how aid works".